language/move-vm/types/src/values/values_impl.rs

```diff
@@ -9,7 +9,9 @@ use move_core_types::{
         words_in, AbstractMemorySize, GasAlgebra, GasCarrier, GasUnits, CONST_SIZE, REFERENCE_SIZE,
         STRUCT_SIZE,
     },
+    transaction_argument::TransactionArgument,
```

Comment:
it was good for `TransactionArgument` not to be included by VM stuff... so overall I would love for this to be out of here

Reply:
Technically this crate already depends on `move-core-types`. This conversion code is a new way to construct values.